src/app.ts

```diff
@@ -1194,21 +1194,115 @@ class App {
       try {
         const accessToken = req.session.accessToken;
         await xero.setTokenSet(accessToken);
+
+        // GET 1099 REPORT
+        // optional parameters
+        const reportYear: string = "2019";
+        const getTenNinetyNineResponse = await xero.accountingApi.getReportTenNinetyNine(req.session.activeTenant, reportYear);
+
+        // getting a contact first
+        const contactsGetResponse = await xero.accountingApi.getContacts(req.session.activeTenant);
+
+        // GET AGED PAYABLES BY CONTACT REPORT
+        // required parameters
+        const apbcContactID: string = contactsGetResponse.body.contacts[0].contactID;
+        // optional parameters
+        const apbcDate: string = "2019-12-31";
+        const apbcFromDate: string = "2019-01-01";
+        const apbcToDate: string = "2019-12-31";
+        const getAgedPayablesByContactResponse = await xero.accountingApi.getReportAgedPayablesByContact(req.session.activeTenant, apbcContactID, apbcDate, apbcFromDate, apbcToDate);
```

Comment:
```suggestion
        const apbcDate = "2019-12-31";
        const apbcFromDate = "2019-01-01";
        const apbcToDate = "2019-12-31";
        const getAgedPayablesByContactResponse = await xero.accountingApi.getReportAgedPayablesByContact(
          req.session.activeTenant,
          apbcContactID,
          apbcDate,
          apbcFromDate,
          apbcToDate
        );
```
So since the function expects typed params, typescript doesn't need us to type out the constants, as they simply won't be expected from the interface.